modules/nw-dual-stack-convert.adoc

```diff
@@ -2,10 +2,12 @@
 [id="nw-dual-stack-convert_{context}"]
 = Converting to a dual-stack cluster network
 
-As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. After converting to a dual-stack networking, new and existing pods have dual-stack networking enabled.
+As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. After converting to a dual-stack network, new and existing pods have dual-stack networking enabled.
 
 Converting a single-stack cluster network to a dual-stack cluster network consists of creating patches and applying them to the cluster's network and infrastructure.
 
+If you need to add IPv6 Virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster, you only need to patch the cluster's infrastructure and not the cluster's network.
```

Comment:
Nit - [per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only), "only" should be placed as close to the phrase it modifies as possible/reasonable:
```suggestion
If you need to add IPv6 Virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster, you need to patch only the cluster's infrastructure and not the cluster's network.
```

Reply:
Thanks, updated.